php.ini-development

```diff
@@ -841,17 +841,17 @@ default_socket_timeout = 60
 ; If you wish to have an extension loaded automatically, use the following
 ; syntax:
 ;
-;   extension=modulename.extension
+;   extension=modulename
 ;
 ; For example, on Windows:
 ;
-;   extension=msql.dll
+;   extension=msql
 ;
 ; ... or under UNIX:
 ;
-;   extension=msql.so
+;   extension=msql
 ;
-; ... or with a path:
+; ... or with an absolute path:
```

Comment:
I'd be inclined to mention that for absolute paths, a file extension must be provided (still), since it now differs to the relative path syntax (and so the aforementioned example on line 844 is not applicable).